library/junos_install_config

```diff
@@ -250,6 +261,9 @@ def _load_via_netconf(module):
         logging.error(msg)
         module.fail_json(msg=msg)
 
+    if args['ignore_warning'] in BOOLEANS + ["True", "False"]:
+        args['ignore_warning'] = module.boolean(module.params['ignore_warning'])
+
```

Comment:
It would be helpful to do additional arg type checking here to ensure that `args['ignore_warning']` is a string type of list of string types.